website/src/content/api-reference/commercetools-frontend-application-components.mdx

```diff
@@ -759,6 +760,22 @@ const DetailPage = () => {
   This is a convenience proxy export to expose pre-defined Intl messages defined in the `@commercetools-frontend/i18n` package.
   The Intl messages can for instance be used for button labels.
 
+## TabHeader
+
+Tab Headers facilitate navigation within a tabular page.
+
+In a tabular page, tabs must be rendered using the `<TabHeader>` component via the `tabControls` prop. A `<TabHeader>` is rendered as a link and it assumes the "tab content" is controlled and rendered using `<Route>` components.
```

Comment:
Perhaps we could also explain that the component _reads_ the current URL path to determine its `active` state.